docs/spec/addresses/bech32.md

```diff
@@ -18,4 +18,4 @@ While all user facing interfaces to Cosmos software should exposed Bech32 interf
 
 To covert between other binary representation of addresses and keys, it is important to first apply the Amino encoding process before Bech32 encoding.
 
-A complete implementation of the Amino serialization format is unnecessary in most cases. Simply prepending bytes from this [table](https://github.com/tendermint/spec/blob/master/spec/blockchain/05-encoding.md#public-key-cryptography) to the byte string payload before Bech32 encoding will sufficient for compatible representation.
+A complete implementation of the Amino serialization format is unnecessary in most cases. Simply prepending bytes from this [table](https://github.com/tendermint/spec/blob/master/spec/blockchain/05-encoding.md#public-key-cryptography) to the byte string payload before Bech32 encoding will sufficient for compatible representation. <!-- Dead Link -->
```

Comment:
not sure what link to use as original is dead

Reply:
Seems like they've deprecated it to another link: https://github.com/cometbft/cometbft/blob/main/spec/blockchain/encoding.md